models.py

```diff
@@ -77,18 +76,6 @@ def lnurl(self, req: Request) -> Lnurl:
         )
         return lnurl_encode(url)
 
-    def lnurl_response(self, req: Request) -> LnurlWithdrawResponse:
```

Comment:
was this not used?

Reply:
yes, also the linter would have caught it